ethcore/account-state/src/state.rs

```diff
@@ -1137,8 +1137,8 @@ impl<B: Backend> State<B> {
 	}
 }
 
-//// TODO: cloning for `State` shouldn't be possible in general; Remove this and use
-//// checkpoints where possible.
+// TODO: cloning for `State` shouldn't be possible in general; Remove this and use
+// checkpoints where possible.
```

Comment:
do we have an issue for this?

Reply:
Not that I know of no. I'm hesitant to add one because I don't fully understand why state should not be cloneable. Do you?

Reply:
it was added in https://github.com/paritytech/parity-ethereum/pull/4632, 
other related PRs
* https://github.com/paritytech/parity-ethereum/pull/2172
* https://github.com/paritytech/parity-ethereum/pull/2387

I assume that using checkpoints instead of `Clone`s is more efficient, since we only store the diff.
We still use clones here https://github.com/paritytech/parity-ethereum/blob/856a0755888a30d4dc52a68cb2638a8bfd5786ac/ethcore/src/block.rs#L239

cc @tomusdrw

Reply:
https://github.com/paritytech/parity-ethereum/issues/11512